web/src/components/projectDelete/DeleteFormSecondConfirmation.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Text, Box, Flex } from 'rebass';
+import FormTitle from '../common/UI/FormTitle';
+import { StyledFormButton } from '../common/UI/Button';
+
+const DeleteFormSecondConfirmation: React.FC<any> = ({ licensePlate, nextPage, previousPage }) => {
+  return (
+    <>
+      <FormTitle style={{ margin: 0, paddingBottom: '20px' }}>
+        Deletion Check for {licensePlate}
```

Comment:
Should be lower case if not the first word, this change would be consistent with the text below
```suggestion
        Deletion check for {licensePlate}
```